subxt/src/backend/unstable/mod.rs

```diff
@@ -194,130 +214,146 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
         keys: Vec<Vec<u8>>,
         at: T::Hash,
     ) -> Result<StreamOfResults<StorageResponse>, Error> {
-        let queries = keys.iter().map(|key| StorageQuery {
-            key: &**key,
-            query_type: StorageQueryType::Value,
-        });
+        retry(|| async {
+            let queries = keys.iter().map(|key| StorageQuery {
+                key: &**key,
+                query_type: StorageQueryType::Value,
+            });
 
-        let storage_items =
-            StorageItems::from_methods(queries, at, &self.follow_handle, self.methods.clone())
-                .await?;
+            let storage_items =
+                StorageItems::from_methods(queries, at, &self.follow_handle, self.methods.clone())
```

Comment:
This doesn't need addressing in this PR, since we currently just emit a Disconnected error to users if the storage stream is interrupted, but a nice improvement would be to tweak `StorageItems` so that if it sees the `DisconnectedWillReconnect` error, it will pick up from where it left off w.r.t streaming storage items if possible (and _not_ restart the whole stream of items again).

This is the same sort of thing I said in an above comment re legacyBackend storage streams :)